contracts/evm/src/base/utils/RollupOperators.sol

```diff
@@ -11,7 +11,7 @@ import {BN254} from "eigenlayer-middleware/src/libraries/BN254.sol";
  * @dev The operator set is an alternative representation of the AVS' original
  * operator set, as it assumes a one-quorum one-weight based voting
  */
-library Operators {
+library RollupOperators {
```

Comment:
```suggestion
library AvsOperators {
```

Those operators both on Rollups and L1 so `RollupOperators` feels a bit misleading.

Reply:
I actually think this makes things a bit more confusing. This is about the operator definition for rollups, since there is another definition for mainnet. `AvsOperators` imo doesn't differentiate between the two. Wdyt?

Reply:
The library is used in both because the operator set update message, which is meant to be passed to rollups, is based on the rollup operators, but it's verifiable in mainnet checkpoints.
We need to differentiate them because even though the meaning is similar the formats and assumptions are different. The operator format for mainnet is determined by the EL core contracts, whereas the rollup operator format is a compact version of it.

Reply:
No, I'm not talking about context of use, I'm talking about actual data formats. There's not the same Operator data format in EL's core contracts. RollupOperators.Operator is indeed only used in the rollup contracts. The way verification is done and such is completely different between both.